config/env.js

```diff
@@ -75,6 +75,8 @@ function getClientEnvironment(publicUrl) {
         // This should only be used as an escape hatch. Normally you would put
         // images into the `src` and `import` them in code to get their paths.
         PUBLIC_URL: publicUrl,
+        // Transak
+        TRANSAK_API_KEY: process.env.TRANSAK_API_KEY,
```

Comment:
As this env variable is used in browser side you can prefix variable with REACT_APP_ ex: `process.env.REACT_APP_TRANSAK_API_KEY`.
This way there is no need to add custom environment keys to config files, can freely use .env files.

Variables without prefix is considered secure, will be used only by node and will not be passed to browser.